connection.go

```diff
@@ -207,25 +210,27 @@ type Opts struct {
 // Address could be specified in following ways:
```

Comment:
Missing dots really seems inconsistent in godoc

http://127.0.0.1:6060/pkg/github.com/tarantool/go-tarantool/#Connect

![image](https://user-images.githubusercontent.com/20455996/161789327-d083d959-e953-4e4b-9938-31d54fbdfe99.png)